### ISSUE TITLE
How to print training/validation loss?

### ISSUE THREAD

**Issue by pasquale90** [Media-Smart/vedadet#72]
During training, a single line for each epoch is printed, nevertheless the loss is not printed. Neither during validation. For the validation I managed to print the metrics (ap,recall,mAP) for each single epoch, using iteratively the validation code provided and then parsing the lines of my custom script's output file, but since I am trying to train on my custom dataset, I would like to obtain a clearer insight of both the training and validation process by printing the losses (either focal ,iou, L1, crossentropy or even the global loss) .. What is the simplest way on retrieving them? Thanks!
